doc/lightning-plugin.7.txt

```diff
@@ -0,0 +1,48 @@
+LIGHTNING-PLUGIN(7)
+===================
+:doctype: manpage
+
+NAME
+----
+lightning-plugin - Manage plugins with RPC
+
+SYNOPSIS
+--------
+*plugin* command [parameter] [second_parameter]
+
+DESCRIPTION
+-----------
+
+The *plugin* RPC command allows to manage plugins without having to restart lightningd.
+It takes 1 to 3 parameters: a command (start/stop/startdir/reload/list) which describes the
```

Comment:
This still calls it `reload` whereas the code below calls it `rescan`

Reply:
Didnt see this one, thanks !